rainier-core/src/main/scala/com/stripe/rainier/compute/LogLineOps.scala

```diff
@@ -24,6 +24,58 @@ private[compute] object LogLineOps {
    */
   def log(@unused line: LogLine): Option[Real] = None
 
+  /*
+  If possible, return a representation of ax as a list of terms to be summed. Within limits, it's helpful here
+  to distribute the multiplications across any internal sums, because that may well surface some terms
+  that are in common with whatever we're about to sum this with.
+   */
+  val DistributeToMaxTerms = 20
+  def distribute(line: LogLine): Option[Line] = {
```

Comment:
what's the O complexity of `distribute`? is it still true that `LogLine` is small?

Reply:
It's O(1), and yes, LogLine should still be small.

Reply:
Gah. I meant to write O(n).